docs/user/general-api.md

```diff
@@ -146,6 +146,13 @@ See the [custom pings documentation](pings/custom.md) about adding custom pings
 
 {{#include ../tab_footer.md}}
 
+## Behavior when uninitialized
+
+Metric recording that happens before Glean is initialized is queued and applied when Glean is initialized.
+To avoid unbounded memory growth the queue is bounded (currently to a maximum of 100 tasks). Further recordings are dropped.
+
+Ping collection and upload as well as the built-in pings are only available after initialization.
```

Comment:
I think this is a bit confusing: you *can* submit a ping prior to initialization, it will just be queued like everything else.

Reply:
Ah, that is true. Though built-in pings are not handled, sending a followup.